build.properties

```diff
@@ -1,2 +1,2 @@
-java_source = 1.8
-java_target = 1.8
+java_source = 17
+java_target = 17
```

Comment:
Is it necessary to raise the target version to 17 too?

I'm fine with raising the source level, but would raise the target level only as far as it's really required.

Reply:
I admit that this is a bit beyond me, I am not sufficiently understanding about the compilation of Java modules.
What is certain is that compilation remains possible with 1.8 as it was.